packages/cli/src/commands/generate/scaffold/__tests__/scaffoldPathMultiwordNoNest.test.js

```diff
@@ -1,11 +1,13 @@
 globalThis.__dirname = __dirname
 import path from 'path'
 
+import { vi, describe, beforeAll, test, expect } from 'vitest'
+
 import '../../../../lib/test'
```

Comment:
This is used for automocking, right? Is that still supported? Do we still need this import?

Reply:
Yeah this is certainly one of the things I'd like to refactor out once we have this one in.